todos/src/components/Todos.js

```diff
@@ -1,35 +1,47 @@
-import React from "react";
+import { React } from "react";
 import AddTask from "components/AddTask";
 import Task from "components/Task";
 import "styles/todos.css";
 import { useSelector } from "react-redux";
 import EmptyTaskList from "components/EmptyTaskList";
+import LoadMoreBtn from "components/LoadMoreBtn";
 
 function Todos() {
   const list = useSelector((state) => state.todoReducers.list);
   const isAddTaskVisible = useSelector(
     (state) => state.toggleReducers.isAddTaskVisible
   );
+  const currentPage = useSelector(
+    (state) => state.currentPageReducer.currentPage
+  );
+  const displayedTodoList = list.slice(0, 4 * currentPage);
```

Comment:
Don't use raw number as you may have to change it sometimes later. Maintain a const in the `constants.js` file use it here

```suggestion
  const displayedTodoList = list.slice(0, PER_PAGE * currentPage);
```